packages/Alert/__tests__/Alert.test.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import expect from 'expect';
 import { shallow } from 'enzyme';
-import Alert from '../index';
+import Alert from '../src/index';
```

Comment:
You don't need to specify `index` with `'../src' will be enough. 
Same for all other components.

Reply:
👍